package.json

```diff
@@ -92,13 +96,18 @@
       "@commitlint/config-conventional"
     ]
   },
+  "workspaces": [
```

Comment:
This is basically symlinking the "proxy" folder to the root projects node_modules, right?